tests/lib/rules/no-todo.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @fileoverview Forbid the use of QUnit.todo
+ * @author Baptiste Doucerain
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-todo"),
+    RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+function createError(callee) {
+    return {
+        messageId: "noQUnitTodo",
+        data: {
+            callee
+        }
+    };
+}
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("no-todo", rule, {
+    valid: [
+        "QUnit.module.test('Name', function() { });",
+        "QUnit.test('Name', function() { });",
+        "module.test('Name', function() { });",
+        "test('Name', function() { });"
+    ],
+
+    invalid: [
+        {
+            code: "QUnit.module.todo('Name', function() { });",
+            errors: [createError("QUnit.module.todo")]
```

Comment:
I'd prefer if we just wrote out the full error here without the helper function. Only a single line anyway:
```js
errors: [{messageId: "noQUnitTodo", data: { callee }]
```

And avoiding unnecessary helper functions reduces indirection and actually makes it easier for eslint-plugin-eslint-plugin to lint our test cases.

Reply:
[Done](https://github.com/platinumazure/eslint-plugin-qunit/pull/218/files?authenticity_token=C8Ze5h3srjvxmteYfvXt4%2BU2TkB8cnkqOm8vr48ODrqm0Y6U19Xxh8r7nvYywQYwr9EUwSTo9f6MLiqAsPoWEw%3D%3D&file-filters%5B%5D=.js#diff-a4cf67dcacaad2b6a3b18d378d94a6deb7fdc27cf576fd6c71c75d7fd2af73d0R36)